index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+    <title>Capstone</title>
+</head>
+<body>
+    <header>
+        <nav class="first-nav">
+        </nav>
```

Comment:
- [ ] I would like to recommend you use a div tag instead of another nav and move this div tag inside of your nav tag below. Please fix it for semantic reason :+1: